accessible_api/AccessibilityRequestHandler.py

```diff
@@ -0,0 +1,124 @@
+""" Service requests for accessible objects """
+
+import sys
+from BaseHTTPServer import BaseHTTPRequestHandler
+from re import search
+from urlparse import urlsplit, parse_qsl
+from json import dumps
+from comtypes import CoInitialize
+from accessible_lib.scripts.accessible import accessible
+from accessible_lib.scripts.event import event
+from accessible_lib.scripts.commands import execute_command
+
+
+class AccessibilityRequestHandler(BaseHTTPRequestHandler):
+    def _successful_response(self, json):
+        self.send_response(200)
+        self.send_header('Content-Type', 'application/json')
+        self.end_headers()
+        self.wfile.write(json)
+
+    def _bad_response(self, message):
+        self.send_response(400, message)
+        self.send_header('Content-Type', 'application/json')
+        self.end_headers()
+
+    def _set_identifiers(self, params):
+        name = params.get('name')
+        role = params.get('role')
+
+        identifiers = {}
+        if name is not None:
+            identifiers["Name"] = name
+        if role is not None:
+            identifiers["Role"] = role
+
+        return identifiers
+
+    def do_GET(self):
```

Comment:
I think we should split this method up, it's a little hard to follow. I would take out the 3 if statement contents into their separate functions. I would call them similarly: get_accessible, get_event and get_cmd. Then you can decoratively match the request path to the method sort of like that (might require some changes to the request API):
```
def do_GET(self):
    url = urlparse.urlparse(self.path)
    handler = getattr(self, "get_" + url.path, None)
    if handler:
        handler(url.query)
    else:
        self.send_response(403, 'Invalid Request')
        self.send_header('Content-Type', 'application/json')
        self.end_headers()
    return
```
Also we should differentiate when the request fails because it's invalid vs when we don't find anything but the request itself is good. Invalid requests should be 400 ones, valid but unsuccessful ones should be 500s (internal server error).